main.tf

```diff
@@ -154,3 +154,13 @@ resource "aws_sns_topic_subscription" "this" {
   subscription_role_arn           = try(each.value.subscription_role_arn, null)
   topic_arn                       = aws_sns_topic.this[0].arn
 }
+
+################################################################################
+# Data Protection Policy
+################################################################################
+
+resource "aws_sns_topic_data_protection_policy" "this" {
+  count  = var.create && var.data_protection_policy != null && !var.fifo_topic ? 1 : 0
```

Comment:
Please add an empty line after meta-arguments like `count` in resources, data-sources